lib/roby/plan.rb

```diff
@@ -1609,6 +1609,17 @@ def finalize_event(event, timestamp = nil)
             event.finalized!(timestamp)
         end
 
+        # Copy plan marks from one into the other, additive only i.e. it does not remove
+        # marks from the destination task
+        #
+        # @param from_task The task whose marks will be copyed
+        # @param to_task The task who will be marked just as replaced_task
+        def copy_marks!(from_task:, to_task:)
```

Comment:
keyword arguments for two arguments is a bit too much to my taste. The convention is `to, from` (like memcpy).

`!` in ruby is an indication of "danger". There's nothing dangerous here.